oop/js/Factorial.js

```diff
@@ -12,6 +12,25 @@ export class Factorial {
   }
 
   // write your code here
+  calculateFactorialUsingLoop() {
```

Comment:
"Factorial" part on this method name is unnecessary, as this class itself already named "Factorial".